proto/Makefile

```diff
@@ -0,0 +1,8 @@
+# Code generated by Mage-tools. DO NOT EDIT.
+.DEFAULT_GOAL := hej
+include ../.mage/tools.mk
+
+.PHONY: hej
+hej: $(mage)
+	@$(mage) proto:hej
+
```

Comment:
Would be nice to avoid outputting this trailing blank line

Reply:
Fixed